ArduCopter/AP_Arming.cpp

```diff
@@ -211,6 +211,14 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
             }
             return false;
         }
+        // Ensure an Aux Channel is configured for motor interlock
+        if (rc().find_channel_for_option(RC_Channel::aux_func_t::MOTOR_INTERLOCK) == nullptr) {
+            if (display_failure) {
+                gcs().send_text(MAV_SEVERITY_CRITICAL,"PreArm: Motor Interlock not configured as Aux Channel Option");
```

Comment:
the STATUSTEXT mavlink message only accepts 50 characters so I think the last 10 characters of this message will be cut off.  Probably best to shorten it to something like "PreArm: Motor Interlock not configured".